titus-api/src/main/java/com/netflix/titus/api/jobmanager/model/job/BasicContainer.java

```diff
@@ -54,18 +54,22 @@
     )
     private final Map<String, String> env;
 
+    private final List<VolumeMount> volumeMounts;
```

Comment:
This class misses validation annotations.

Reply:
Added `CollectionInvariants`